tests/compile-fail/tracked_fn_incompatibles.rs

```diff
@@ -34,4 +34,30 @@ fn tracked_fn_with_too_many_arguments_for_specify(
 ) -> u32 {
 }
 
+#[salsa::interned]
+struct MyInterned<'db> {
+    field: u32,
+}
+
+#[salsa::tracked]
+fn tracked_fn_with_lt_param_and_elided_lt_on_db_arg<'db>(
```

Comment:
Can you add a variant of this test where the lifetime is named `'db1` or something like that, to show how that is reflected in the error message?

Reply:
Sure